stdlib/src/builtin/string.mojo

```diff
@@ -1453,7 +1467,7 @@ fn _toggle_ascii_case(char: Int8) -> Int8:
 
 fn _calc_initial_buffer_size_int32(n0: Int) -> Int:
     # See https://commaok.xyz/post/lookup_tables/ and
-    # https://lemire.me/blog/2021/06/03/computing-the-number-of-digits-of-an-integer-even-faster/
+    # https://lemire.me/blog/2021/06/03/computing-the-number-of-DIGITS-of-an-integer-even-faster/
```

Comment:
I believe that would break the URL here, the "search and replace" might have been a bit too quick 😄

Reply:
That was accidental by using `change all occurences` instead of normal renaming in my IDE.
Good catch, thx!